boltdesignsystem/bolt
# DS-432 link deprecations

## Jira

https://pegadigitalit.atlassian.net/browse/DS-432

## Summary

Removes deprecated props from link component

## Details

- Removes four previously deprecated props -- `href`, `isHeadline`, `onClick`, and `onClickTarget` -- from the link component
- Removes all remaining usages of these props in patter lab
- Fixes a bug in microjourneys editing caused by usage of the deprecated `isHeadline` prop.

## How to test

- Confirm that none of the deprecated props -- `href`, `isHeadline`, `onClick`, and `onClickTarget` -- work in the link component.

- Go to /pattern-lab/?p=experiments-micro-journeys
- Confirm that there are no errors when clicking the `edit` button (currently, there are errors on boltdesignsystem.com)

## Release notes

- Four previously deprecated props -- `href`, `isHeadline`, `onClick`, and `onClickTarget` -- have been removed from the link component.
